c_src/dpiStmt_nif.c

```diff
@@ -127,6 +127,7 @@ DPI_NIF_FUN(stmt_getQueryValue)
     return map;
 }
 
+#include "dpiQueryInfo_nif.h"
```

Comment:
Is this standard practice in `C` ? shouldn't be included at the top instead ?

Reply:
> Is this standard practice in C ?

Don't know of any.

> shouldn't be included at the top instead ?

I can move it to top. In this case it don't matter, where it is included.